scale-tests/scale_test.py

```diff
@@ -22,35 +22,43 @@
 
 
 def submit_job(dispatcher):
+    print("T dispatcher: {}".format(dispatcher))
     dispatcher_name, dispatcher_role, driver_role = dispatcher
 
-    args = ["--conf", "spark.cores.max=1",
+    args = ["--conf", "spark.cores.max=4",
+            "--conf", "spark.executor.cores=1",
             "--conf", "spark.mesos.containerizer=mesos",
-            "--conf", "spark.mesos.role={}".format(driver_role),
-            "--conf", "spark.mesos.executor.docker.image=mesosphere/spark-dev:931ca56273af913d103718376e2fbc04be7cbde0",
+            "--conf", "spark.mesos.executor.docker.image=mesosphere/spark-dev:f5dd540adffd9ab9e3e826e48d22e39ebc296567-aee120236ee37309892ddbee42acf0028096947f",
             # use Hector's image
-            "--conf", "spark.port.maxRetries=32"  # setting to allow up to 32 drivers on same node
+            "--conf", "spark.port.maxRetries=32",  # setting to allow up to 32 drivers on same node
             #"--conf", "spark.mesos.driverEnv.SPARK_USER=root", # Run as root on centos
+            "--supervise",
+            #"--conf", "spark.mesos.rejectOfferDuration=1000s",
+            "--conf", "spark.mesos.driver.failoverTimeout=30",
+            "--conf", "spark.mesos.task.labels=jpmc_test_id:spark_batch_run001"
             ]
 
-    app_args = "100000 300"
+    app_args = "100000 900" # runs for about an hour
 
     utils.submit_job(
         app_name="/{}".format(dispatcher_name),
         app_url=MONTE_CARLO_APP_URL,
         app_args=app_args,
         verbose=False,
-        args=args)
+        args=args,
+        driver_role=driver_role)
 
 
 def submit_loop(launch_rate_per_min, dispatchers):
+    print("dispatchers: {}".format(dispatchers))
```

Comment:
Is there a reason to use `print` instead of logging. I see that we're already using it elsewhere in the file, so this is probably out of scope for this PR.

Reply:
I'll replace it with logging.